.github/workflows/tester.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'npm'
       - name: Install Dependencies
-        run: npm install
+        run: npm install --force
```

Comment:
Why use `--force`?

Reply:
Install Dependencies get error on Github Action.

but when i tried it locally i don't get any error, i used same version nodejs.

Reply:
i add --force after get solution from a website